src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -252,7 +252,7 @@ protected string GetStringQueryString(string name, bool required = true)
             return val[0];
         }
 
-        private static void ThrowRequiredMember(string name)
+        protected static void ThrowRequiredMember(string name)
```

Comment:
Can be made private after using `GetQueryStringValueAndAssertIfSingleAndNotEmpty`

Reply:
See other comments.